docs/design-documents/20220309-opencdc.md

```diff
@@ -0,0 +1,349 @@
+# OpenCDC
+
+## Introduction
+
+### Goals
+
+The main goal is to ensure compatibility between any combination of source/destination connectors.
+
+Define a format that fulfills the following criteria:
+- A record can be associated to a schema that describes the payload or key
+- The paylod/key schema can be included in the record or pointing to a schema stored in a schema registry
+- Be able to represent an inserted, updated and deleted record
+- Be able to represent a record created in a snapshot
+- Support for splitting big records into multiple parts
```

Comment:
I don't see this mentioned again in this document. Is there more to this or do you consider this achievable with just metadata and record payloads?

Reply:
I forgot to mention it in the standard metadata chapter, I was thinking of having some standard metadata field to denote multipart records. I'll add this!